src/Views/AppInfoView.vala

```diff
@@ -737,7 +737,37 @@ namespace AppCenter.Views {
 
             uninstall_button.clicked.connect (() => uninstall_clicked.begin ());
 
-            realize.connect (load_more_content);
+            realize.connect (() => {
```

Comment:
This looks like my commit, but rather than cherry-picking, it is just copy pasted. That isn't a correct rebase.

Reply:
There was an indentation error which I corrected (a section of the code was indented an extra level, which is not correct for our code style), which I think is why it looks like the code block was copy-pasted. The only actual change in this block other than whitespace is on line 744/765

Reply:
Ya, small changes can be made with a rebase, and then the commit will have multiple authors (which is fine, that's just how git works), but a rebase should give credit to the correct authors. I am not sure if other code in this rebase has the author removed like this, but we should always provide credit

Reply:
What's the commit you're referencing specifically?

Reply:
https://github.com/pop-os/shop/commit/2bc8c7d87b0d1df92d0649d80e21e3c84ef82148

Reply:
`git branch --contains 2bc8c7d87b0d1df92d0649d80e21e3c84ef82148` lists that commit as being in this branch

Reply:
Then the commit shouldn't be `fix:screenshot scaling` it should be `fix indentation`, and those changes should be listed, because this is more than just a rebase then

Reply:
I should mention that the update was technically performed using a merge rather than a rebase, if that makes a difference

Reply:
@13r0ck I've pushed commits which should correct what you're mentioning, let me know if that works